WebService/ComparisonEngineUI/ComparisonEngineUI/ViewModels/MapViewModel.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using ComparisonEngineUI.Views;
+using Xamarin.Forms;
```

Comment:
There should be an empty space between using and namespace

Reply:
Fixed